### DISCUSSION TITLE
I can't use

### DISCUSSION THREAD

**Discussion by Aloneinthedarkwed** [thatskymod/Sky-CotL-Scripts#121] [Q&A]
Dukun langit inactive, smite always crash, roman alway stop at 3%

**Comment by thatskymod** [comment]
Hello,

several mods have been updated to match the latest Sky update 0.26.5. Kindly try to apply the updated mods once again and ensure `Bypass game updates` is turned **OFF** in Mod Manager.

Let me know if this helps!

**Comment by thatskymod** [comment]
I can infer that this thread relates to an earlier version of Sky, 0.26.5.

Upon inspection, it was determined that the following versions are compatible and operational:
- libLangit043.so (Live + Beta)
- libRomanBeta.so 2.1.0 (Live only)

Outdated and incompatible versions have been removed from this collection for the sake of clarity.
